include/RequestResponse/task_request.h

```diff
@@ -21,11 +26,16 @@ class TaskRequest : public Payload {
     void setLeaderUuid(const std::string& leaderUuid);
     void setAssignedWorkers(const AddressTable& assignedWorkers);
     void setTrainingData(const std::vector<int>& trainingData);
+    void setTrainingFile(const std::string& trainingFile);
+    void setTrainingDataFromFile();
+
+    void createTrainingFile(const std::string& newTrainingFileName);
```

Comment:
`createTrainingFile` should only be run once. would it make more sense to make this a private function and have it be called on `TaskRequest` initialization?

Reply:
Good idea  👍🏼